data-collection/data-collection/Utilities/ImagePickerPermissions.swift

```diff
@@ -148,6 +145,10 @@ public struct ImagePickerPermissions {
                         alert.addAction(settingsAction)
                     }
                     
+                    let okAction = UIAlertAction.okay()
```

Comment:
Can you name the variable `okayAction`?
```suggestion
                    let okayAction = UIAlertAction.okay()
```

Reply:
Yep. I'll change it.